src/routes/note-view.test.js

```diff
@@ -0,0 +1,96 @@
+import { MemoryRouter } from "react-router-dom";
+import { render, cleanup, act, fireEvent } from '@testing-library/react';
+
+import NoteViewRoute from './note-view';
+import Harness from '../../test/jest/helpers/harness';
+
+jest.mock('@folio/stripes/smart-components', () => ({
+  NoteViewPage:
+    ({
+      navigateBack,
+      onEdit,
+    }) => (
+      <>
+        <button
+          type="button"
+          onClick={navigateBack}
+        >
+          navigateBack
+        </button>
+
+        <button
+          type="button"
+          onClick={onEdit}
+        >
+          onEdit
+        </button>
+      </>
+    )
+}));
+
+const history = {
+  goBack: jest.fn(),
+  push: jest.fn(),
+};
+
+const location = {
+  pathname: 'pathname',
+  search: '?searchType=notes&q=test&offset=1',
+  hash: '',
+};
+
+const match = {
+  params: {
+    noteId: 'noteId',
+  },
+};
+
+const getNoteViewRoute = (props = {}) => (
+  <MemoryRouter>
+    <Harness>
+      <NoteViewRoute
+        history={history}
+        location={location}
+        match={match}
+        {...props}
+      />
+    </Harness>
+  </MemoryRouter>
+);
+
+const renderNoteViewRoute = (props) => render(getNoteViewRoute(props));
+
+describe('Given NoteViewRoute', () => {
+  beforeEach(() => {
+    history.goBack.mockClear();
+    history.push.mockClear();
+  });
+
+  afterEach(cleanup);
+
+  it('should render NoteViewRoute', async () => {
+    await act(async () => {
+      await renderNoteViewRoute();
+    });
+  });
+
+  describe('when when click on navigateBack button', () => {
+    it('should navigate back', () => {
```

Comment:
You test here a case when you don't have `location.state`, so I suggest to rename the test case:
```suggestion
    it('should redirect to eholdings page', () => {
```

But you also have a case with `location.state` present and there it should handle "history.goBack" and you can add a test for it too.